pr-deploy/test/app.test.ts

```diff
@@ -14,14 +14,12 @@
  * limitations under the License.
  */
 
-// mock unzipAndMove dependency before importing '../src/app'
+// mock decompressAndMove dependency before importing '../src/app'
 jest.mock('../src/zipper', () => {
   return {
-    unzipAndMove: jest
+    decompressAndMove: jest
```

Comment:
Is this the only non-formatting change in this file? Hard to tell, help

Reply:
Yep, only non-formatting change